### PR TITLE
refactor: Move tests to specific command folder

### PR DESCRIPTION
**Pull request by hosekpeter**
Jira: [PSGO-455](https://keboola.atlassian.net/browse/PSGO-455)

**Changes:**
- moved tests to specific command folder
- renamed package

-----------

[PSGO-455]: https://keboola.atlassian.net/browse/PSGO-455?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: internal/pkg/service/cli/cmd/ci/workflow/workflows_test.go [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
 
-	f := workflow.DefaultFlags()
+	// terminal
+	console, err := terminal.New(t)
+	require.NoError(t, err)
+
+	p := cli.NewPrompt(console.Tty(), console.Tty(), console.Tty(), false)
+
+	// dialog
+	d := dialog.New(p, o)
```
Comment (michaljurecko) [inline internal/pkg/service/cli/cmd/ci/workflow/workflows_test.go:31]: Please, replace this duplicate code with a common function, for example `dialog.NewForTest(...)`.

Comment (michaljurecko) [reply internal/pkg/service/cli/cmd/ci/workflow/workflows_test.go:31]: @hosekpeter Is this fixed? 

image

Comment (hosekpeter) [reply internal/pkg/service/cli/cmd/ci/workflow/workflows_test.go:31]: @michaljurecko yes -> https://github.com/keboola/keboola-as-code/pull/1626/commits/fa9432750726a8f54da6e2c82434a07f2bf49f0a

Comment (michaljurecko) [reply internal/pkg/service/cli/cmd/ci/workflow/workflows_test.go:31]: :man_shrugging: And also `TestAskWorkflowsOptionsInteractive` function?

image

Comment (hosekpeter) [reply internal/pkg/service/cli/cmd/ci/workflow/workflows_test.go]: done

**Review by michaljurecko — APPROVED** [review]
LGTM

**Comment by michaljurecko** [comment]
Please, fix CS.

image

**Pull request merged**